docs/catalog/cloud/use-optimal-software-patterns-for-data.md

```diff
@@ -0,0 +1,48 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: cloud
+tags: 
+ - compute
+ - role:cloud-engineer
+ - size:medium
+---
+
+# Use optimal software patterns and architectures for data access and storage
+
+## Description
+
+Application generally need to work with data and every application has specific usage patterns which in turn define in which patterns data is accessed. Optimizing the storage for these type of patterns or optimizing the access to the storage to an optimal pattern can lead to lower carbon usage.
+
+
+## Solution
+
+Monitor and analyze the application to identify if the data access or storage technology is optimal for the use cases of the application. Optimize the access or switch to a more optimal storage technology.
```

Comment:
I'll leave this discussion for the SME review but I'm wondering if the level of this pattern is too high of a level and these potential solutions should be their own patterns.

Reply:
Agreeing with Bill that I think the below should be split out into separate patterns.

Reply:
So a 
"Use optimal software patterns and architectures for data **access**"
and a 
"Use optimal software patterns and architectures for data **storage**"
?

Reply:
Not quite! What you have shared above is still one intent mapping to multiple solutions. I think what we are looking for here is a 1:1 mapping. A starting point can be, why is it more sustainable to use in-memory/key-value-like data for session storage?